ui/src/clientLibraries/components/ClientCSharpOverlay.tsx

```diff
@@ -17,155 +29,48 @@ const ClientCSharpOverlay: FunctionComponent<{}> = () => {
         </a>
       </p>
       <br />
-      <h5>Installing package</h5>
+      <h5>Installing Package</h5>
       <p>
         <b>Package Manager</b>
       </p>
-      <pre>
-        <code>Install-Package InfluxDB.Client</code>
-      </pre>
+      <CodeSnippet
+        copyText={installingPackageManagerCodeSnippet}
+        label="Code"
+      />
       <p>
         <b>.NET CLI</b>
       </p>
-      <pre>
-        <code>dotnet add package InfluxDB.Client</code>
-      </pre>
+      <CodeSnippet
+        copyText={installingPackageDotNetCLICodeSnippet}
+        label="Code"
+      />
       <p>
         <b>Package Reference</b>
       </p>
-      <pre>
-        <code>&lt;PackageReference Include="InfluxDB.Client"/&gt;</code>
-      </pre>
+      <CodeSnippet copyText={packageReferenceCodeSnippet} label="Code" />
       <h5>Initializing the Client</h5>
-      <pre>
-        <code>
-          using InfluxDB.Client;
-          <br />
-          <br />
-          namespace Examples
-          <br />
-          &#123;
-          <br />
-          &nbsp;&nbsp;public class Examples
-          <br />
-          &nbsp;&nbsp;&#123;
-          <br />
-          &nbsp;&nbsp;&nbsp;&nbsp;public static void Main(string[] args)
-          <br />
-          &nbsp;&nbsp;&nbsp;&nbsp;&#123;
-          <br />
-          &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;var client =
-          InfluxDBClientFactory.Create("basepath", "token".ToCharArray());
-          <br />
-          &nbsp;&nbsp;&nbsp;&nbsp;&#125;
-          <br />
-          &nbsp;&nbsp;&#125;
-          <br />
-          &#125;
-          <br />
-        </code>
-      </pre>
-      <h5>Using the client to execute a query</h5>
-      <pre>
-        <code>
-          const string query = "from(bucket: \"my_bucket\") |> range(start:
-          -1h)";
-          <br />
-          <br />
-          var tables = await client.GetQueryApi().QueryAsync(query,
-          "someorgid");
-        </code>
-      </pre>
+      <CodeSnippet copyText={initializeClientCodeSnippet} label="C# Code" />
       <h5>Writing Data</h5>
       <p>
-        Data could be written using InfluxDB Line Protocol, Data Point or POCO
-      </p>
-      <p>
-        <b>InfluxDB Line Protocol</b>
+        <b>Option 1: Example for writing using InfluxDB Line Protocol</b>
```

Comment:
~Option 1: Example for writing using InfluxDB Line Protocol~ -> Option 1: Use InfluxDB Line Protocol to write data